core/src/main/java/de/danoeh/antennapod/core/sync/SyncService.java

```diff
@@ -112,19 +126,16 @@ public Result doWork() {
     public static void clearQueue(Context context) {
         executeLockedAsync(() ->
                 context.getSharedPreferences(PREF_NAME, Context.MODE_PRIVATE).edit()
-                    .putLong(PREF_LAST_SUBSCRIPTION_SYNC_TIMESTAMP, 0)
-                    .putLong(PREF_LAST_EPISODE_ACTIONS_SYNC_TIMESTAMP, 0)
-                    .putLong(PREF_LAST_SYNC_ATTEMPT_TIMESTAMP, 0)
-                    .putString(PREF_QUEUED_EPISODE_ACTIONS, "[]")
-                    .putString(PREF_QUEUED_FEEDS_ADDED, "[]")
-                    .putString(PREF_QUEUED_FEEDS_REMOVED, "[]")
-                    .apply());
+                        .putLong(PREF_LAST_SUBSCRIPTION_SYNC_TIMESTAMP, 0)
+                        .putLong(PREF_LAST_EPISODE_ACTIONS_SYNC_TIMESTAMP, 0)
+                        .putLong(PREF_LAST_SYNC_ATTEMPT_TIMESTAMP, 0)
+                        .putString(PREF_QUEUED_EPISODE_ACTIONS, "[]")
+                        .putString(PREF_QUEUED_FEEDS_ADDED, "[]")
+                        .putString(PREF_QUEUED_FEEDS_REMOVED, "[]")
+                        .apply());
     }
 
     public static void enqueueFeedAdded(Context context, String downloadUrl) {
-        if (!GpodnetPreferences.loggedIn()) {
-            return;
-        }
```

Comment:
Removing this means we will get an ever-growing list of actions when someone does not use sync. Could lead to a slow/unresponsive app.

Reply:
i'll replace this with a condition checking if there is an active sync service

Reply:
done